students/seoyun/westagram/users/views.py

```diff
@@ -38,4 +38,21 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message": "KEY_ERROR"}, status=400)
 
+class LogInView(View):
+    def post(self, request):
+        try:
+            user_data  = json.loads(request.body)
+            email      = user_data['email']
+            password   = user_data['password']
+            login_user = User.objects.get(email=email)
+            
+            if not (password == login_user.password):
```

Comment:
```suggestion
            if not User.objects.filter(email=email, password=password):
```
이런식으로 `filter()` 메소드를 사용하여 여러 조건을 한 번에 해결할 수도 있습니다.
현재 상황에서 이렇게 저렇게 여러 방식으로 시도해보신 것 같아 보기 좋네요~!